packages/xo-web/src/common/xo/index.js

```diff
@@ -1141,7 +1141,12 @@ export const deleteTemplates = templates =>
     await Promise.all(
       map(resolveIds(templates), id =>
         _call('vm.delete', { id }).catch(reason => {
-          if (vmIsTemplate.is(reason)) {
+          const template = getObject(store.getState(), id)
+          if (
+            forbiddenOperation.is(reason) &&
+            template.isDefaultTemplate &&
+            !('destroy' in template.blockedOperations)
```

Comment:
But by testing the message directly, it can make some issues of maintainability

Reply:
Then @julien-f's suggestion is even better since the test becomes trivial:
```js
throw incorrectState({
  object: vmId,
  property: 'isDefaultTemplate',
  actual: true,
  expected: false,
})
```
and the test:
```js
incorrectState.is(error, { property: 'isDefaultTemplate', expected: false })
```
The property should never change and the test doesn't rely on a message.